### DISCUSSION TITLE
Use the DefaultApi.GetHeartsAsync()  API to get the model list without license information

### DISCUSSION THREAD

**Discussion by DavidWeng916** [pixiv/vroid-sdk-developers#98] [Q&A] [answered]
Hello, I would like to ask you, I added this [model](https://hub.vroid.com/en/characters/5537810530425756564/models/1913629511074924621) to Heart,
then I use **DefaultApi.GetHeartsAsync()** API to get the list of hearts,
but there is no license information for this model (CharacterModel.license is empty).
I would like to ask if this is normal?
Because without license information, there is no way to display user terms.

**Comment by nyamadan** [comment, marked as answer]
This model is the VRM1.0 model.
For VRM1.0 models, CharacterModel.license is not provided.
Use the [vrm_meta](https://developer.vroid.com/sdk/docs/0.2.2/Pixiv.VroidSdk.Api.DataModel.CharacterModelVersion.html#Pixiv_VroidSdk_Api_DataModel_CharacterModelVersion_vrm_meta) field instead.
We implemented how to handle vrm_meta in CharacterModelDetailRenderer.
I hope this helps you.

**Reply by DavidWeng916** [reply]
I see. Thank you very much for your answer.
